core/panic.c

```diff
@@ -28,6 +28,10 @@
 #include "panic.h"
 #include "arch/panic_arch.h"
 
+#if DEVELHELP && defined MODULE_PS
```

Comment:
quick grepping shows that DEVELHELP is only ever defined in the makefiles, and that we use "#if DEVELHELP", "#ifdef DEVELHELP" and "#if defined(DEVELHELP)" mixed.